### ISSUE TITLE
Proposal: Creator Economy Dapp on Kaia

### ISSUE THREAD

**Issue by afrokayie** [kaiachain/kaia-dapp-mono#311]
Description
- I am building a creator economy dapp called Alpha on Kaia
- **Alpha** is a platform for Web3 creators and Key Opinion Leaders (KOLs) to monetize their knowledge, tools, and time.  

Creators can:  
- Build profiles showcasing curated lists of tools they use for success in the Web3 space.  
- Offer their time for booking consultations or mentorship sessions with payments accepted in KAIA.

Each tool or time slot can have:  
- A price tag in KAIA
- A unique shareable link for easy marketing across social media platforms.  

Bounty track
- Surf Szn

**Comment by zxstim** [comment]
approved. keep it simple at first for your MVP. a reference is [time.fun](https://time.fun)

**Comment by afrokayie** [comment]
Hi, here are the updates and the next milestones:

- Dora Hacks Build link: https://dorahacks.io/buidl/20675
- Github link: https://github.com/eobikoru/Alpha

- Deployed Version: https://alpha-9r4z.vercel.app/

In this first version, we created:

- landing page
- Creator profile and dashboard
- Creator tool adding functionality 

Next Milestone, we will create:

- Consultation functionality for creators 
- Ability to add minute/hourly rate for consultation in Kaia
- Buyer (non-creator) UI

Future milestones, we will create:

- Full-fledged functionality for both creators and buyers 
- Onboard creators in batches 
- Market to and board both crypto natives and noobs.

**Comment by afrokayie** [comment]
@zxstim I'll appreciate your feedback.

**Comment by zxstim** [comment]
- i can't create my profile nor access the dashboard
- github repo is not accessible (returns 404)
<img width="1113" alt="image" src="https://github.com/user-attachments/assets/3821a007-4d38-4ee3-b825-dcfed88494a1" />

**Comment by afrokayie** [comment]
- I apologize for the inconvenience caused.
- The bug has been fixed and the project has been redeployed here: https://alpha-gz2v.vercel.app/
- The Github link is now publicly accessible: https://github.com/eobikoru/Alpha

**Comment by afrokayie** [comment]
. @zxstim Apologies for the inconvenience.

- Here’s the deployed live dApp: https://alpha-v2-psi.vercel.app/

- Here’s the GitHub repo: https://github.com/eobikoru/alpha-v2

- Smart contract code here: https://github.com/eobikoru/alpha-v2/tree/SM/alphaBack/contracts

- Dora Hacks Build link: https://dorahacks.io/buidl/20675




In this session, we have: 

- Improved UI
- Creator profile creation functionality
- Ability to list tools (pdfs, videos etc) and price in KAIA
- Ability to set consultation time and price in KAIA
- Ability to share profile for direct consultation 

Next milestone:

- Add buyer (audience) side functionalities 
- Onboard at least 50 creators

**Comment by zxstim** [comment]
Approved for `1520 KAIA`. Keep improving the dapp to get adjusted bounty amount.

**Comment by zxstim** [comment]
closing

**Comment by afrokayie** [comment]
.@zxstim New Update for Creator Platform (Alpha):

- Buyer or audience side Added
- Ability to explore current creators 
- Ability to buy tools from creators 

To interact:
- Connect with a new wallet address 
- Choose "I'm a buyer option"
- Explore the current creators 

Next Update:

- Ability to book an appointment 
- Analytics for creator Dashboard 

Details: 

- Here’s the deployed live dApp: https://alpha-v2-psi.vercel.app/

- Here’s the GitHub repo: https://github.com/eobikoru/alpha-v2

- Smart contract code here: https://github.com/eobikoru/alpha-v2/tree/SM/alphaBack/contracts

- Dora Hacks Build link: https://dorahacks.io/buidl/20675

**Comment by zxstim** [comment]
make new proposal for the update. there is an "Update" template

**Comment by afrokayie** [comment]
---
name: Update for Creator Economy Dapp on Kaia
about: 
title: Buyer or audience UI integration
labels: ''
assignees: zxstim

---

Description
- **Buyer or audience side integrated**  
- Ability to explore current creators  
- Ability to buy tools from creators  


Reference issue
- #311

Category
- Surf Szn

**Comment by zxstim** [comment]
No. Make a new issue. Each issue is tied with a payout for tracing. So make a new issue.
